docs/comparison.md

```diff
@@ -17,15 +17,15 @@ Feature/Capability Key:
-| First-class TypeScript support                    | ✅                                            | ❓                             | ❓                                     | ❓                                               | ❓                                     |
-| Fully Inferred TypeScript (Including Deep Fields) | ✅                                            | ❓                             | ❓                                     | ❓                                               | ❓                                     |
-| Headless UI components                            | ✅                                            | ❓                             | ❓                                     | ❓                                               | ❓                                     |
-| Framework agnostic                                | ✅                                            | ❓                             | ❓                                     | ❓                                               | ❓                                     |
+| First-class TypeScript support                    | ✅                                            | ❓                             | ❓                                     | ✅                                               | ❓                                     |
+| Fully Inferred TypeScript (Including Deep Fields) | ✅                                            | ❓                             | ❓                                     | ✅                                               | ❓                                     |
+| Headless UI components                            | ✅                                            | ❓                             | ❓                                     | ✅                                               | ❓                                     |
+| Framework agnostic                                | ✅                                            | ❓                             | ❓                                     | 🛑                                               | ❓                                     |
 | Granular reactivity                               | ✅                                            | ❓                             | ❓                                     | ❓                                               | ❓                                     |
-| Nested object/array fields                        | ✅                                            | ✅                             | ❓                                     | ❓                                               | ❓                                     |
+| Nested object/array fields                        | ✅                                            | ✅                             | ❓                                     | ✅                                               | ❓                                     |
```

Comment:
If I could put a tooltip here, I'd say while it suppors nested arrays but `useFeildArray` supports one level nesting

Reply:
You can use a *(1) callout with a corresponding footnote

Reply:
I initially thought it was unsupported, but after rechecking the documentation, it appears to be a TypeScript issue.

> for nested field array, you will have to cast the field array by its name.
>```ts
>const { fields } = useFieldArray({ name: `test.${index}.keyValue` as 'test.0.keyValue' });
>```
https://react-hook-form.com/docs/usefieldarray

Reply:
That's the `*` I'm hoping for - type casting is an obtuse way of supporting this that we don't require

Reply:
See [`99a07ff` (#673)](https://github.com/TanStack/form/pull/673/commits/99a07fff483cfdfcb8e5d223faf4a8e5393a6999)